entities/src/test/java/org/odk/collect/entities/LocalEntityUseCasesTest.kt

```diff
@@ -184,31 +184,6 @@ class LocalEntityUseCasesTest {
         assertThat(songs[0].branchId, equalTo(offline.branchId))
     }
 
-    @Test
-    fun `updateLocalEntitiesFromServer overrides offline version if the online version is the same`() {
```

Comment:
@lognaturel and I chatted about this, and we actually want to do the opposite so that the entity "metadata" and data are always in sync. We also want to make sure that `trunkVersion` and `branchId` make sense in cases where the server has different data, but the same version. I'll make changes to this.